RecoVertex/PrimaryVertexProducer/interface/PrimaryVertexProducer.h

```diff
@@ -94,7 +101,11 @@ class PrimaryVertexProducer : public edm::stream::EDProducer<> {
   edm::EDGetTokenT<reco::TrackCollection> trkToken;
   edm::EDGetTokenT<edm::ValueMap<float> > trkTimesToken;
   edm::EDGetTokenT<edm::ValueMap<float> > trkTimeResosToken;
+  edm::EDGetTokenT<edm::ValueMap<float> > trackMTDTimeQualityToken;
 
-  bool f4D;
-  bool weightFit;
+  bool useTransientTrackTime;
+  bool useMVASelection;
```

Comment:
here and in quite a few other files these updates adding new/renamed members are a good time to start using a common (and recommended) pattern of naming data members with a trailing underscore